templates/terraform/resource.html.markdown.erb

```diff
@@ -82,6 +83,23 @@ To get more information about <%= object.name -%>, see:
 
 ~> **Warning:** <%= object.docs.warning -%>
 <%- end -%>
+<%- if !sensitive_props.empty? -%>
+<%-
+  sense_props = sensitive_props.map! {|prop| "`"+prop.name.underscore+"`"}.uniq
+  if sense_props.length == 1
```

Comment:
I think we have access to ActiveSupport here, so [`to_sentence`](https://apidock.com/rails/Array/to_sentence) should work!

Reply:
oh nice! i saw that as an option but was afraid i'd have to install something new.  thanks!